build.gradle

```diff
@@ -65,6 +65,26 @@ configurations {
     integrationTestRuntime.extendsFrom testRuntime
 }
 
+def getV8Dependency() {
+    def arch = System.getProperty('os.arch')
+    def dep = null
+    if (OperatingSystem.current().isWindows()) {
+        dep = 'win32_x86'
+        if (arch.equals('amd64')) {
+            dep += '_64'
+        }
+    } else if (OperatingSystem.current().isMacOsX() && (arch.equals('amd64') || arch.equals('x86_64'))) {
+        dep = 'macosx_x86_64'
+    } else if (OperatingSystem.current().isLinux() && arch.equals('amd64')) {
+        dep = 'linux_x86_64'
+    }
+    if (dep == null) {
+        logger.error("Could not find V8 runtime compatible to this system")
+        return null
```

Comment:
Seeing that (i) the current rendering is very slow and (ii) others will surely use that library too, citeproc natively in Java is a huge win for the community. 

I fear, however, that it will be a huge effort and we would also need that effort in JabRef itself.